### DISCUSSION TITLE
Link Keyword/Pages to Google Search Console

### DISCUSSION THREAD

**Discussion by goaround** [towfiqi/serpbear#72] [Ideas]
It would be helpful to link a Keyword/Country/Page in the Discover & Insight tab to the Google Search Console e.g. for a page: `https://search.google.com/search-console/performance/search-analytics?resource_id=https://my-domain.com/&breakdown=query&page=*/my-page/&metrics=CLICKS,IMPRESSIONS,POSITION` Just replace `https://my-domain.com/` and `*/my-page/`

**Comment by towfiqi** [comment]
Thanks for the suggestion. As you may have noticed, it has been already fixed in version 0.2.5

**Comment by goaround** [comment]
Hey @towfiqi,

I can't find a link to the Google Search Console in the Discover & Insight tab in v0.2.6.
